pandas/tests/window/test_grouper.py

```diff
@@ -69,9 +68,10 @@ def test_rolling(self):
             expected = g.apply(lambda x: getattr(x.rolling(4), f)(ddof=1))
             tm.assert_frame_equal(result, expected)
 
-        result = r.quantile(0.5)
-        expected = g.apply(lambda x: x.rolling(4).quantile(0.5))
-        tm.assert_frame_equal(result, expected)
+        for f in ["linear", "lower", "higher", "midpoint", "nearest"]:
```

Comment:
Either here or in a follow up could you parametrize this test with these values rather than the loop within?

Reply:
can  u make this a separate test and parmetrize

Reply:
Yes, I think I can handle that here - working on it now.

Reply:
I wrote a new test called `test_rolling_quantile` and parametrized the `interpolation` argument as requested, but unfortunately it looks as though most of the tests in this file have broken since I merged in master.  To test, I checked out the upstream master branch, rebuilt my conda environment from scratch, and ran the tests there.  10 of the tests failed, including `test_rolling` (stack trace below).  Is this a known issue?

`self = <pandas.tests.window.test_grouper.TestGrouperGrouping object at 0x116432208>

    def test_rolling(self):
        g = self.frame.groupby("A")
        r = g.rolling(window=4)

        for f in ["sum", "mean", "min", "max", "count", "kurt", "skew"]:

>           result = getattr(r, f)()

test_grouper.py:64:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
../../core/window/rolling.py:1870: in sum
    return super().sum(*args, **kwargs)
../../core/window/rolling.py:1245: in sum
    return self._apply("roll_sum", "sum", **kwargs)
../../core/window/common.py:81: in _apply
    return self._groupby.apply(f)
../../core/groupby/groupby.py:718: in apply
    result = self._python_apply_general(f)
../../core/groupby/groupby.py:734: in _python_apply_general
    keys, values, mutated = self.grouper.apply(f, self._selected_obj, self.axis)
../../core/groupby/ops.py:171: in apply
    result_values, mutated = splitter.fast_apply(f, group_keys)
../../core/groupby/ops.py:904: in fast_apply
    return libreduction.apply_frame_axis0(sdata, f, names, starts, ends)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

>   ???
E   AttributeError: 'Int64Index' object has no attribute '_has_complex_internals'

pandas/_libs/reduction.pyx:519: AttributeError
`

Reply:
Oops, @WillAyd's suggestion to rebuild the Cython extensions was correct.  The tests are now passing; I'm going to push my changes shortly.

Reply:
The change is pushed - let me know if that's different from what you had in mind!